dogfooding/lib/screens/call_participants_list.dart

```diff
@@ -0,0 +1,129 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_dogfooding/widgets/share_call_card.dart';
+import 'package:stream_video_flutter/stream_video_flutter.dart';
+
+class CallParticipantsList extends StatefulWidget {
+  const CallParticipantsList({
+    super.key,
+    required this.call,
+  });
+
+  final Call call;
+
+  @override
+  State<CallParticipantsList> createState() => _CallParticipantsListState();
+}
+
+class _CallParticipantsListState extends State<CallParticipantsList> {
+  StreamSubscription<CallState>? _callStateSubscription;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _callStateSubscription = widget.call.state.listen(_setState);
```

Comment:
Why don't we wrap this `call.state` in a builder widget and drop this `StatefulWidget` to `StatelessWidget`?

Reply:
Good idea, done